### ISSUE TITLE
Cursor Speed

### ISSUE THREAD

**Issue by poliveira89** [prydonius/spinning_cursor#2]
It is possible to increase or decrease speed of cursor?

The "animation"/"effect" it is too slow for my needs.. in some cases almost doesn't work as i wanted, because it's slow..

**Comment by prydonius** [comment]
Hey there,

At the moment there's no option to specify the speed, however I will add this in soon.

Meanwhile, you can override the cursor like here:
https://gist.github.com/3042533

Hope this helps

**Comment by prydonius** [comment]
This feature has now been implemented in ae81c2d. A 'delay' option was added that can be used just like all the other options e.g.

``` ruby
SpinningCursor.start do
    action do
        sleep 2
    end
    message "Enjoying the spin?"
    delay 0.1
end
```
